docs/developer/payments/refunds.mdx

```diff
@@ -291,3 +303,39 @@ Below is an example order with multiple transactions and `totalRemainingGrant` a
   It also means that `totalChargedAmount` includes `amountChargePending`, and `totalRefundedAmount` includes `amountRefundPending`.
 - Authorized amounts, including `amountAuthorized` and `amountAuthorizePending`, are also considered processed. These amounts will contribute to the overall processed totals and impact the `overchargedAmount`.
   :::
+
+
+
+
+## Refund settings
+
+:::note
+Available from 3.22
+:::
+
+While `reason` field is optional and remains a plain text field, the `reasonReference` can be configured:
+- Require staff user to pass `reasonReference` for all refunds.
+- Make `reasonReference` optional
+
+To change the settings, you can either use the Dashboard or a mutation
+
+```graphql
+mutation UpdateRefundSettings($id: ID) {
+    refundSettingsUpdate(input: {
+      reasonReferenceType: $id
+    }) {
+        reasonReferenceType
+    }
+}
+```
```

Comment:
You should also attach the example variables and show the response of this mutations. Use `Tab` for it. You can see the examples here: https://github.com/saleor/saleor-docs/pull/1674